tests/fixtures/sdk_code/feature_operating_system.py

```diff
@@ -0,0 +1,20 @@
+# Generated by SDK version: {sdk_version}
+from bson import ObjectId
+from featurebyte import SCDTable
+
+scd_table = SCDTable.get_by_id(ObjectId("{table_id}"))
+scd_view = scd_table.get_view(
+    view_mode="manual",
+    drop_column_names=["is_active"],
+    column_cleaning_operations=[],
+)
+scd_view["os_type"] = "unknown"
+col = scd_view["os_type"]
+scd_view["os_type"][col.str.contains(pat="window", case=True)] = "window"
+col_1 = scd_view["os_type"]
+scd_view["os_type"][col.str.contains(pat="mac", case=True)] = "mac"
+grouped = scd_view.as_features(
+    column_names=["os_type"], feature_names=["os_type"], offset=None
+)
+feat = grouped["os_type"]
+output = feat
```

Comment:
For the statement like
```
col_1 = scd_view["os_type"]
```
We could use `vulture` package (https://github.com/jendrikseipp/vulture) to detect & remove unused statements.

Reply:
Seems like internally, `vulture` is using `ast`. Will update the SDK code generation logic in the next PR to remove unused statements.